src/components/common/Textarea.jsx

```diff
@@ -0,0 +1,19 @@
+import * as S from "./Textarea.styles";
+
+const Textarea = ({ value, label, error, placeholder, onChange, id }) => {
```

Comment:
💬 여담
저의 개인적인 취향에서는 props를 우선순위 혹은 필수값이 위에 오게 작성하는 것을 좋아해요. 이 기준은 모두에게 다르겠지만.. 개인적으로는 아래와 같은 순서를 좋아합니다~
지훈님도 한번 고민해보시면 좋을 것 같아요.


```suggestion
const Textarea = ({ 
  id, 
  value, // id, value 가 가장 중요한 값이라고 생각해서 최상단으로 위치
  label, 
  error, 
  placeholder, 
  onChange // onChange는 유일한 함수라서 최하단에 위차
}) => {
```